sky/skylet/providers/lambda_cloud/lambda_utils.py

```diff
@@ -86,8 +87,10 @@ def raise_lambda_error(response: requests.Response) -> None:
         resp_json = response.json()
         code = resp_json.get('error', {}).get('code')
         message = resp_json.get('error', {}).get('message')
-    except (KeyError, json.decoder.JSONDecodeError):
-        raise LambdaCloudError(f'Unexpected error. Status code: {status_code}')
+    except json.decoder.JSONDecodeError:
+        raise LambdaCloudError('Unexpected error. Status '
+                               f'code: {status_code}; response: {response}; '
+                               f'resp_json: {resp_json}')
```

Comment:
Shall we add `response.reason` or `response.text` to indicate which error happened? Since repr of `requests.Response` is something like `<Response [404]>` and does not contain a lot of useful information.

Also, why are we printing `resp_json` in `except json.decoder.JSONDecodeError` ? IIUC the `resp_json` must be `None`?

Reply:
Good catch, done.